doc/edge-metrics-v2.md

```diff
@@ -0,0 +1,43 @@
+# IoT Edge Metrics
+
+## IoT Edge Hub
+
+The IoT Edge Hub module exposes a number of metrics in the Prometheus exposition format that provide insights into its operational state.
+
+### How to enable
+
+As of release 1.0.9, metrics are exposed as an experimental feature available at http port **9600** of the Edge Hub module. To enable, the following environment variables should be set for the module (make note of the double underscores):
+
+| Environment Variable Name                | value  |
+|------------------------------------------|--------|
+| `ExperimentalFeatures__Enabled`          | `true` |
```

Comment:
Just fyi - I found out that experimentalFeatures:Enabled as well. But maybe we should stick to __ for now.